packages/website/src/features/Deploy/SafeAddressInput.tsx

```diff
@@ -56,42 +58,43 @@ export function SafeAddressInput() {
 
   // Load the safe address from url
   useEffect(() => {
-    if (searchParams.address || searchParams.chainId) {
-      const chainId = searchParams.chainId;
-      const address = searchParams.address;
-      const newSafe = parseSafe(`${chainId}:${address}`);
+    const { safeAddress: address, chainId } = searchParams;
+
+    if (isNaN(Number(chainId))) return;
```

Comment:
On this case is the same, but I usually prefer the `isSafeInteger` check, as it also checks for undefined, weird floats, etc
```suggestion
    if (!Number.isSafeInteger(Number(chainId))) return;
```